BondageClub/Scripts/Typedef.d.ts

```diff
@@ -123,6 +123,73 @@ declare function LoseFocus(event: MouseEvent): void;
 
 //#endregion
 
+//#region Chat
+
+interface ChatRoom {
+	Name: string;
+	Description: string;
+	Admin: number[];
+	Ban: number[];
+	Limit: number;
+	Game: any;
```

Comment:
Assured by server code
```suggestion
	Game: string;
```